pennylane/templates/subroutines/adder.py

```diff
@@ -22,54 +22,83 @@
 class Adder(Operation):
     r"""Performs the in-place modular addition operation.
 
-     This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
-     computational basis:
+    This operator performs the modular addition by an integer :math:`k` modulo :math:`mod` in the
+    computational basis:
 
-     .. math::
+    .. math::
 
-         \text{Adder}(k, mod) |x \rangle = | x+k \; \text{modulo} \; mod \rangle.
+        \text{Adder}(k, mod) |x \rangle = | x+k \; (mod) \rangle.
 
-     The implementation is based on the quantum Fourier transform method presented in
-     `arXiv:2311.08555 <https://arxiv.org/abs/2311.08555>`_.
+    This operation can be represented in a quantum circuit as:
```

Comment:
I am not sure adding the image here is helpful. The equation is clear enough.

Reply:
I would leave it if possible. Although in the equation it is clear, it seems to me more visual the in-place / out-place difference seen in this way